spec/views/forms/check_your_answers/show.html.erb_spec.rb

```diff
@@ -73,6 +78,10 @@
     it "displays the email field" do
       expect(rendered).to have_field(I18n.t("helpers.label.email_confirmation_form.confirmation_email_address"))
     end
+
+    it "contains a hidden notify reference for the confirmation email" do
+      expect(rendered).to have_css("input", id: "confirmation-email-reference", visible: :hidden)
```

Comment:
Instead of only looking for the hidden field we should test it has the value we are expecting to see.
```suggestion
      expect(rendered).to have_field("confirmation-email-reference", type: "hidden", with: email_confirmation_form.confirmation_email_reference)
```